scenarios/kubernetes_e2e.py

```diff
@@ -575,7 +574,7 @@ def main(args):
     if args.kubeadm:
         version = kubeadm_version(args.kubeadm, shared_build_gcs_path)
         runner_args.extend([
-            '--kubernetes-anywhere-path=/workspace/kubernetes-anywhere',
```

Comment:
format substitution would be more consistent with the following lines

Reply:
done.